dtfs-central-api/Dockerfile

```diff
@@ -9,9 +9,12 @@ ENV GITHUB_SHA=$GITHUB_SHA
 WORKDIR /app
 COPY package.json .
 COPY package-lock.json .
-RUN npm ci
+
+COPY libs libs
+COPY dtfs-central-api dtfs-central-api
+
+RUN npm run ci:all
```

Comment:
Should this be not `npm run ci` as I am only clean installing container in question here?